content/en/blog/2022/metrics-announcement.md

```diff
@@ -0,0 +1,35 @@
+---
+title: OpenTelemetry Metrics Release Candidates
+date: 2022-05-18
+author: Morgan McLean
+draft: true # TODO: remove this line once your post is ready to be published
+---
+
+OpenTelemetry’s metrics capabilities are now available as release candidates! This means that the specification, APIs, SDKs, and other components that author, capture, process, and otherwise interact with metrics now have the full set of OpenTelemetry metrics functionality and are ready for use. These release candidates will be promoted to general availability throughout the next few weeks.
+
+The 1.0 metrics release includes the following:
+
+- Metrics functionality included in the OpenTelemetry APIs for every GA-declared language, which provide language-specific interfaces that can create and manipulate metrics, and associate metadata and attributes to each. These are useful for:
```

Comment:
This is a little confusing coming before the next major point, which indicates that there are no GA-declared languages. The examples given still ultimately end up depending on a given language's support, I think?

Reply:
Good point. The APIs are mostly GA while the SDKs are not (even for a given language), but I don't think that I got this across very clearly. Fixing.

Reply:
```suggestion
- Metrics functionality included in the OpenTelemetry language-specific APIs, which provide language-specific interfaces that can create and manipulate metrics, and associate metadata and attributes to each. These are useful for:
```